deploy/examples/create-external-cluster-resources.py

```diff
@@ -826,7 +829,44 @@ def validate_rados_namespace(self):
             raise ExecutionFailureException(
                 ("The provided rados Namespace, '{}', is not found in the pool '{}'").format(
                     rados_namespace, rbd_pool_name))
-
+    
+    def get_rgw_fsid(self):
+        access_key = self.out_map['RGW_ADMIN_OPS_USER_ACCESS_KEY']
+        access_key = "WC2D69F9L7KEOJVETKER"
+        secret_key = self.out_map['RGW_ADMIN_OPS_USER_SECRET_KEY']
+        secret_key = "3iBk9hrYydYNd5QURzG8kXHo20muJXATPC2Xtpzf"
+        http_request = "GET"
+        rgw_endpoint = self._arg_parser.rgw_endpoint
+        resource = "/admin/info"
+        query_string = "format=json"
+        contentType = "application/x-compressed-tar"
+        dateTime = datetime.datetime.utcnow()
+        dateTimeFormat = dateTime.strftime('%a, %d %b %Y %X +0000')
+        headerToSign = http_request + "\n" + "\n" + contentType + "\n" + dateTimeFormat + "\n" + resource
+        print(headerToSign)
+        cmd = "echo -en " + headerToSign + " | openssl sha1 -hmac " + secret_key +" -binary | base64"
```

Comment:
let's be careful if `openssl` does not exist on the machine. Alternatively, it will be good to see Python has something we can use as a lib instead.